app/assets/stylesheets/_user_journey.scss

```
@@ -36,3 +36,9 @@
 dt {
   display: block;
 }
+.govuk-error-summary--grey {
+  border-color: $govuk-border-colour;
+}
+.govuk-error-summary--orange {
+  border-color: #CB4B0B;
```

my preference would be
```suggestion
  border-color: govuk-colour("orange");
```
It's slightly lighter, but makes me more comfortable using pre-defined set of colours instead of introducing new colours...

---

Orange is a lot brighter, this is copied from the prototype which is the same colour as the expiry tag we have

---

in that case can we have the colour defined once as a variable and use the variable for both the tag and the border here?